src/resources/LibraryStyles.css

```diff
@@ -397,7 +398,6 @@ input {
 .SearchResultItemContainer .ItemIcon {
     padding: 2px 8px;
     width: 40px;
-    height: 40px;
```

Comment:
I see, I haven't seen the description text yet! Could you clarify why `.ItemInfo` width needs to be set as a relative percentage? The problem with relative sizing with an absolute-sized icon will arise when the user tries to resize the window/library view.